### PR TITLE
fix non-pointer error in client get

### PR DESCRIPTION
**Pull request by sickhye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Arbitrary structs are converted to JSON when they are `put` into a region:
     }
     
     v := &MyStruct{"Joe", 42}
-    client.Put("Joe", v)    
+    client.Put("REGEION", "Joe", v)
```
Comment (jdeppe-pivotal) [inline README.md]: Please would you correct the spelling to `REGION`.

File: README.md [hunk 2]
```diff
@@ -59,12 +59,12 @@ Arbitrary structs are converted to JSON when they are `put` into a region:
...
 
 Similarly, to retrieve the data:
 
     v := &MyStruct{}
-    x := client.Get("Joe", v)
+    x := client.Get("REGEION", "Joe", v)
```
Comment (jdeppe-pivotal) [inline README.md]: ...and the same here.

**Comment by sickhye** [comment]
@jdeppe-pivotal 
sorry, I Fixed typo in Readme

**Comment by jdeppe-pivotal** [comment]
Thanks for the fix!

**Pull request merged**